apps/passport-client/build.ts

```diff
@@ -207,3 +215,14 @@ function compileHtml(): void {
 
   fs.writeFileSync(path.join("public", "index.html"), html);
 }
+
+function copyGPCArtifacts(): void {
+  fs.rmSync(path.join("public/artifacts/proto-pod-gpc"), {
+    recursive: true
+  });
```

Comment:
this part cause me an error when the path does not exist

Reply:
Interesting.  I can see why that happens, though I wonder why the CI build in a clean environment didn't run into this.  Fix incoming...

Reply:
https://github.com/proofcarryingdata/zupass/pull/1722

Reply:
yeah, thank you. I just try to pull the latest commit, and run into the error